esbuild.js

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is it intentional that this PR doesn't include removing the old webpack configs it obsoletes?

Reply:
Yes ... but I was torn. Ultimately, it's probably best to remove them (they're always there in the history if we want to revert or something) but figured that could be a future PR, after adding the report package which would completely omit the need for any web extension related configs.